.github/workflows/main.yml

```diff
@@ -86,3 +86,34 @@ jobs:
         uses: pypa/gh-action-pypi-publish@release/v1
         with:
           password: ${{ secrets.PYPI_API_TOKEN }}
+
+
+  trigger_endless-key-app:
+    runs-on: ubuntu-latest
+    if: startsWith(github.ref, 'refs/tags/') || startsWith(inputs.tagname, 'v')
+    needs: [build_apps-bundle, build_kolibri-explore-plugin_whl]
+
+    steps:
+      - name: Get tag name as the release version when at a tag
+        if: startsWith(github.ref, 'refs/tags/')
+        run: |
+          echo "VERSION=${{ github.ref_name }}" >> $GITHUB_ENV
+
+      - name: Get tag name as the release version from input
+        if: ${{ !startsWith(github.ref, 'refs/tags/') && startsWith(inputs.tagname, 'v') }}
+        run: |
+          echo "VERSION=${{ inputs.tagname }}" >> $GITHUB_ENV
+
+      - name: Trigger building endless-key-app
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{ secrets.BUILD_ENDLESSKEYAPP_TOKEN }}
```

Comment:
Per https://github.com/endlessm/eos-administration/commit/e53ba80fd359645f1cb3017f46339c438e2dafb2, I decided to name the secret `REPOSITORY_DISPATCH_TOKEN` since it's not actually scoped to the endless-key-app repo but really to the `eos-shell-build` user.